docs/getting-started/gamefi-sdk.md

```diff
@@ -0,0 +1,389 @@
+        string contract = "contract_address";
+        string tokenId = "token_id";
+        string rpc = "https://evm-cronos.crypto.org";
+
+        string ownerOf = await ERC721.OwnerOf(chain, network, contract, tokenId, rpc);
+
+        if (ownerOf == account) 
+        {
+            // fetch uri from chain
+            string uri = await ERC721.URI(chain, network, contract, tokenId, rpc);
+            print("uri: " + uri);
+
+            // fetch json from uri
+            UnityWebRequest webRequest = UnityWebRequest.Get(uri);
+            await webRequest.SendWebRequest();
+            Response data = JsonUtility.FromJson<Response>(System.Text.Encoding.UTF8.GetString(webRequest.downloadHandler.data));
+
+            // parse json to get image uri
+            string imageUri = data.image;
+            print("imageUri: " + imageUri);
+
+            // fetch image and display in game
+            UnityWebRequest textureRequest = UnityWebRequestTexture.GetTexture(imageUri);
+            await textureRequest.SendWebRequest();
+            this.gameObject.GetComponent<Renderer>().material.mainTexture = ((DownloadHandlerTexture)textureRequest.downloadHandler).texture;
+        }
+    }
+}
+```
+::: tip NOTE
+  We are using `PlayerPrefs.GetString("Account")` to dynamically fetch the address when connecting to Metamask. You can replace this with the actual account address for testing purposes like so: `string account = "account_address"`
+:::
+
+### Build the Scenes
+To build the Game Login example, you need to go to `File/Build Settings` and select your WebGL, IOS or Androi Build Settings. In this example, we will build using the WebGL method.
+
+<img src="./assets/gameFi/build_settings.png" />
+
+::: tip NOTE
+  Make sure to select the right WebGL Template in the Player -> Resolution and Presentation settings.
+:::
+
+On successfull completion, you will be able to login to your Metamask wallet. On succesfull login, the scene will load any textures from NFT assets you own. 
+
+## Scripts
+
+### Player Account
+
+PlayerPrefs.GetString("Account") is the user's wallet account accessed after the LoginScene.
+
+```csharp
+string account = PlayerPrefs.GetString("Account");
+print(account);
+```
+
+### EVM 
+
+Get the current latest block number
+```csharp
+string chain = "cronos";
+string network = "mainnet";
+string rpc = "https://evm-cronos.crypto.org";
+
+int blockNumber = await EVM.BlockNumber(chain, network, rpc);
+print(blockNumber);
+```
+
+Get the balance of the native blockchain
+```csharp
+string chain = "cronos";
+string network = "mainnet";
+string account = "account_address";
+string rpc = "https://evm-cronos.crypto.org";
+
+string balance = await EVM.BalanceOf(chain, network, account, rpc);
+print(balance);
+```
+
+Verify a signed message.
+```csharp
+string message = "YOUR_MESSAGE";
+string signature = "0x94bdbebbd0180195b89721a55c3a436a194358c9b3c4eafd22484085563ff55e49a4552904266a5b56662b280757f6aad3b2ab91509daceef4e5b3016afd34781b";
```

Comment:
Maybe change the sample signature to some string to be filled?

Reply:
done :)